src/libData/AccountData/AccountStoreSCEvm.cpp

```diff
@@ -254,15 +237,23 @@ uint64_t AccountStoreSC<MAP>::InvokeEvmInterpreter(
 template <class MAP>
 bool AccountStoreSC<MAP>::ViewAccounts(EvmCallParameters& params, bool& ret,
                                        std::string& result) {
-  TransactionReceipt rcpt;
   uint32_t evm_version{0};
-  evmproj::CallResponse response;
+  evmproj::CallResponse response{};
+
+#if 0
```

Comment:
Here we have two problems:
1. The eth_call doesn't have anything to do with the transaction, so even the fact that TransactionReceipt is involved indicates that calling through EvmCallRunner is superfluous here.
2. Reintroducing multithreading, even though via std::async might mean reintroducing the problem of the crash again (since we don't know exactly why it was crashing, and it was one of the hypotheses).

@CSideSteve PTAL